cmd/determinize-ci-operator/main.go

```diff
@@ -53,3 +92,52 @@ func main() {
 		}
 	}
 }
+
+func migrateOpenshiftInstallerSRCTemplates(
+	configuration *config.DataWithInfo,
+	allowedBranches sets.String,
+	allowedOrgs sets.String,
+	allowedCloudproviders sets.String,
+) (migratedCount int) {
+	if (len(allowedBranches) != 0 && !allowedBranches.Has(configuration.Info.Branch)) || (len(allowedOrgs) != 0 && !allowedOrgs.Has(configuration.Info.Org)) {
+		return 0
+	}
+
+	for idx, test := range configuration.Configuration.Tests {
+		if test.OpenshiftInstallerSrcClusterTestConfiguration == nil ||
+			(len(allowedCloudproviders) != 0 && !allowedCloudproviders.Has(string(test.OpenshiftInstallerSrcClusterTestConfiguration.ClusterProfile))) {
+			continue
+		}
+
+		clusterProfile := test.OpenshiftInstallerSrcClusterTestConfiguration.ClusterProfile
+		test.OpenshiftInstallerSrcClusterTestConfiguration = nil
+		test.MultiStageTestConfiguration = &api.MultiStageTestConfiguration{
+			ClusterProfile: clusterProfile,
+			Test: []api.TestStep{{LiteralTestStep: &api.LiteralTestStep{
+				As:       "test",
+				From:     string(api.PipelineImageStreamTagReferenceSource),
+				Commands: test.Commands,
+				Cli:      api.LatestReleaseName,
+				Resources: api.ResourceRequirements{
+					Requests: api.ResourceList{"cpu": "100m"},
```

Comment:
Is this what the template sets right now?

Reply:
It doesn't set anything at all, so it was a pretty random guess: ci-operator/templates/openshift/installer/cluster-launch-installer-src.yaml